app/controllers/api/v2/report_templates_controller.rb

```diff
@@ -115,15 +116,23 @@ def export
       def generate
         @composer = ReportComposer.from_api_params(params)
         if @composer.valid?
-          response = @composer.render(params: params)
-          send_data response, type: @composer.mime_type, filename: @composer.report_filename
+          response.headers['Content-Type'] = @composer.mime_type
+          # https://discuss.rubyonrails.org/t/actioncontroller-live-chokes-at-buf-push/76490
+          response.headers['Cache-Control'] = 'no-cache'
+          response.headers['Last-Modified'] = '0'
+          response.headers['ETag'] = nil
+          response.stream.write("")
+          # TODO  @composer.report_filename
+          @composer.render(output: response.stream, params: params)
```

Comment:
Perhaps instead of writing directly to the steam, we can let rails handle it by passing an Enumerator to `response_body` similar to how we do it in https://github.com/theforeman/foreman/blob/develop/app/controllers/concerns/foreman/controller/csv_responder.rb and https://github.com/theforeman/foreman/blob/develop/app/services/csv_exporter.rb?